client/src/components/landing/components/LandingTop.js

```diff
@@ -35,8 +39,18 @@ function LandingTop({ page }) {
             <AppleLogo />
             <GoogleLogo />
             <MicrosoftLogo />
-            <AmazonLogo />
-            <SpotifyLogo />
+
+            {clientLocale === 'chinese' ? (
```

Comment:
Do we want to show these for `chinese-traditional`, too? cc: @ahmadabdolsaheb

Reply:
I forgot about Chinese-traditional

Reply:
No worries.  I assume it should appear for both, but Ahmad will know :)

Reply:
I believe it should, yes.

Reply:
yes for both.